config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] This definition exposes unnecessary routes. But we don't want that. We only need to expose the **index** and **show** as that is all we need at this point. You can fix this by changing to this

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
done